2025-01-21T14:10:32.3801580Z ##[group]Run pytest tardis --tardis-regression-data=/Users/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append -m "not continuum"
2025-01-21T14:10:32.3802880Z [36;1mpytest tardis --tardis-regression-data=/Users/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append -m "not continuum"[0m
2025-01-21T14:10:32.3932560Z shell: /bin/bash -l {0}
2025-01-21T14:10:32.3932830Z env:
2025-01-21T14:10:32.3933030Z   CACHE_NUMBER: 0
2025-01-21T14:10:32.3933720Z   PYTEST_FLAGS: --tardis-regression-data=/Users/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append
2025-01-21T14:10:32.3934340Z   CODECOV_TOKEN: 
2025-01-21T14:10:32.3934550Z   MAMBA_ROOT_PREFIX: /Users/runner/micromamba
2025-01-21T14:10:32.3934960Z   MAMBA_EXE: /Users/runner/micromamba-bin/micromamba
2025-01-21T14:10:32.3935300Z   CONDARC: /Users/runner/work/_temp/setup-micromamba/.condarc
2025-01-21T14:10:32.3935730Z ##[endgroup]
2025-01-21T14:10:32.6391450Z /Users/runner/work/_temp/c588b690-4c42-4a2d-b5a4-240939408cfa.sh: line 1: pytest: command not found
2025-01-21T14:10:32.6424720Z ##[error]Process completed with exit code 127.
